css/buttons.css

```diff
@@ -0,0 +1,34 @@
+.btn-container {
+	display: flex;
+	justify-content: space-around;
+}
+
+.btn {
+	margin-top: 1rem;
+	display: inline-block;
+	padding: .8rem 2rem;
+	font-size: 1.4rem;
+	color: #fff;
+	background: var(--primary);
+	border-radius: .5rem;
+	cursor: pointer;
+}
+
+
+.btn:hover {
+	background: var(--primary-dark);
+}
+
+.btn-secondary {
+	margin-top: 1rem;
+	display: inline-block;
+	padding: .8rem 2rem;
+	font-size: 1.4rem;
+	color: #fff;
+	background: var(--secondary);
+	border-radius: .5rem;
+	cursor: pointer;
```

Comment:
properties are same for `.btn` and `.btn-secondary`. You can use a comma separator for the same properties (like .btn, .btn-secondary { ..... }) and separately define `background` property.

Reply:
Correct i will implement this using comma as you said✅